README.md

```diff
@@ -5,6 +5,15 @@
 
 This is a client library for interacting with the idam application.
 
+## Note
+
+This repository is not supported by the CFT IDAM team.
```

Comment:
But it isn't supported by us. We didn't publish, we don't own it, and there's no information about who should be approached for problems with it, so it comes back to us. I'm happy for whatever wording you want to add to indicate who will support teams who are having problems using this? Or are you saying that you want IDAM to take it on?

Reply:
Even your above comment contains details that aren't reflected anywhere else.

Reply:
How about "For support please create issues in this repository"?

Reply:
>  but I think the argument would be that a library like this would belong with Software Engineering

@banderous any thoughts? Personally I think clients should belong to the API maintainer, it's in your interest for it to be maintained well, and simplifies integration.

Reply:
perfect. I'll update